web/sites/default/local.services.yml

```diff
@@ -5,7 +5,7 @@
 parameters:
   http.response.debug_cacheability_headers: false
   twig.config:
-    debug: true
+    debug: false
```

Comment:
We don't want to turn this off on local.

Reply:
Oops, I turned that off for my BDS Contacts work so that I could get clean markup in my json feed, but it somehow got saved in this branch. Will fix.